src/adapter/etl-adapter-google-sheet/src/Flow/ETL/Adapter/GoogleSheet/GoogleSheetLoader.php

```diff
@@ -0,0 +1,136 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Flow\ETL\Adapter\GoogleSheet;
+
+use Flow\ETL\{Adapter\GoogleSheet\RowsNormalizer\EntryNormalizer, FlowContext, Loader, Row\Entry, Rows};
+use Google\Service\Sheets;
+use Google\Service\Sheets\ValueRange;
+
+final class GoogleSheetLoader implements Loader, Loader\Closure
+{
+    private ?int $columnsCount = null;
+
+    private string $dateFormat = 'Y-m-d';
+
+    private string $dateTimeFormat = \DateTimeInterface::ATOM;
```

Comment:
not sure about google sheets but excel prefer `Y-m-d H:i:s` over ATOM (`Y-m-d\TH:i:sP`) and it automatically detects it as a date time